src/main/java/catchMultipleExeptions/Demo.java

```diff
@@ -0,0 +1,21 @@
+package catchMultipleExeptions;
+
+import java.util.Scanner;
+
+public class Demo {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter numerator:");
+        int numerator = scanner.nextInt();
+        System.out.println("Enter denominator:");
+        int denominator = scanner.nextInt();
+
+        try {
+            new Arithmetic();
```

Comment:
Same thing here: we don't need to create an instance of a class to call a static method